recipes/sklearn-contrib-lightning/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "sklearn-contrib-lightning" %}
+{%set version = "0.2.0" %}
+{%set hash_type = "md5" %}
+{%set hash_val = "cf373cf22e9c1b0c11133589d3b14926" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - cython
+  run:
+    - python
+
+test:
+  requires:
+    - numpy
+    - scikit-learn
+    - nose
+  imports:
+    - lightning
+
+about:
+  home: http://contrib.scikit-learn.org/lightning/
+  license: BSD
```

Comment:
[`setup.py`](https://github.com/scikit-learn-contrib/lightning/blob/master/setup.py#L18) says `New BSD` and that is the `BSD 3-Clause` if I recall correctly. Please confirm that and modify it here to reflect the right BSD version.